src/actions/transferActions.js

```diff
@@ -259,6 +266,7 @@ async function _transactionHashRetrieved (txRequest: {|
   transferAmount: StandardTokenUnit,
   transferFiatAmountSpot: string,
   fiatType: string,
+  exchangeRate: { cryptoExchangeRate: string, txFeeCryptoExchangeRate: string },
```

Comment:
Will need to make changes to the backend as well?

Reply:
Will need to make changes to the backend as well?

Reply:
already pushed in previous backend pr